src/main/java/com/cleanroommc/groovyscript/compat/mods/pyrotech/Barrel.java

```diff
@@ -26,7 +26,7 @@ public RecipeBuilder recipeBuilder() {
         return new RecipeBuilder();
     }
 
-    @MethodDescription(type = MethodDescription.Type.ADDITION, example = @Example("'iron_dirt_water_to_lava', ore('ingotIron'), ore('ingotIron'), item('minecraft:dirt'), item('minecraft:dirt'), fluid('water'), fluid('lava'), 1000"))
+    @MethodDescription(type = MethodDescription.Type.ADDITION, example = @Example("'iron_dirt_water_to_lava', ore('dye'), ore('dye'), item('minecraft:dirt'), item('minecraft:dirt'), fluid('water'), fluid('lava'), 1000"))
```

Comment:
instead of changing all of these cases, go to `examples/postInit/custom/vanilla.groovy` and comment out line 193 and 194 (`oredict.clear('plankWood')` and `oredict.removeAll('ingotIron')`)